src/hearings/containers/request-hearing/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.spec.ts

```diff
@@ -0,0 +1,39 @@
+import {CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {provideMockStore} from '@ngrx/store/testing';
+import {of} from 'rxjs';
+import {initialState} from '../../../hearing.test.data';
+import {ACTION} from '../../../models/hearings.enum';
+import {HearingsService} from '../../../services/hearings.service';
+import {HearingActualAddEditSummaryComponent} from './hearing-actual-add-edit-summary.component';
+
+describe('HearingViewEditSummaryComponent', () => {
+  let component: HearingActualAddEditSummaryComponent;
+  let fixture: ComponentFixture<HearingActualAddEditSummaryComponent>;
+  const mockedHttpClient = jasmine.createSpyObj('HttpClient', ['get', 'post']);
+  const hearingsService = new HearingsService(mockedHttpClient);
+  hearingsService.navigateAction$ = of(ACTION.CONTINUE);
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [HearingActualAddEditSummaryComponent],
+      providers: [
+        provideMockStore({initialState}),
+        {provide: HearingsService, useValue: hearingsService},
+      ],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(HearingActualAddEditSummaryComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  afterEach(() => {
+    fixture.destroy();
+  });
+});
```

Comment:
this should be moved to the actuals-hearing folder instead of put it in the request hearing folder.